src/components/organisms/OakHeaderHero/OakHeaderHero.tsx

```diff
@@ -45,9 +45,9 @@ const UnstyledComponent = (props: OakHeaderHeroProps) => {
   } = props;
   return (
     <OakBox
-      $overflowX={"hidden"}
       $width={"100%"}
       $background={"bg-decorative3-main"}
+      $overflowX={"hidden"}
```

Comment:
hmm i think this should be moved back where it was

Reply:
😂